docs/CaliptraIntegrationSpecification.md

```diff
@@ -403,9 +403,33 @@ The SoC communicates with the mailbox through an AXI Interface. The SoC acts as
 
 The AXI_USER bits are used by the SoC to identify which device is accessing the mailbox.
 
+## External Staging Area
+
+To save SRAM area when Caliptra operates in Subsystem mode, the mailbox (MBOX) SRAM is reduced to **16 KiB**.  
+Instead of passing images directly to Caliptra through the mailbox, the SoC can configure an external staging SRAM that Caliptra fetches from and processes.
+
+Caliptra Core receives the base address of this staging area through the **SOC_IFC** register `EXTERNAL_STAGING_AREA_ADDRESS`.  
```

Comment:
Should we clarify that the base address should be an AXI address that is accessible via the DMA controller?

Reply:
Done.